src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6914,6 +6914,13 @@ public sealed partial class RuntimeWrappedException : System.Exception
         public object WrappedException { get { throw null; } }
         public override void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
     }
+    [AttributeUsage(AttributeTargets.Module | AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Event, Inherited = false)]
```

Comment:
These should be fully qualified, e.g. `System.AttributeUsage` and `System.AttributeTargets.Module`. The class should also be partial.

I believe that generating the `System.Runtime` reference assembly is still a manual process for the most part (CC. @safern), but its still good to avoid the conflicts for when someone does auto-generate it later.

Reply:
For System.Runtime you can add `--follow-typeforwards` parameter to the `GenAPI` call in the target that we have in order to generate the reference assembly. 

Basically just append it here:https://github.com/dotnet/runtime/blob/6afe96cc2e950379d13a87efa1fa5a651cbce9bd/src/libraries/Directory.Build.targets#L132

Then you can call `dotnet msbuild /t:GenerateReferenceSource` in the `System.Runtime` src project and it should update `ref/System.Runtime.cs`. 

System.Runtime.cs has a lot of manual updates so there might be some other stuff changed that you'll need to ignore.

Reply:
I tried this, and the build failed with

```
C:\Users\angocke\code\runtime\src\libraries\Directory.Build.targets(151,5): error MSB3073: The command " "C:\Users\angocke\code\runtime\artifacts\obj\System.Reflection.TestModule\netstandard2.0-Debug\System.Reflection.TestModule.dll" --lib-path "C:\Users\angocke\code\runtime\artifacts\bin\ref\netstandard2.0" --out "C:\Users\angocke\code\runtime\src\libraries\System.Runtime\tests\ref\System.Reflection.TestModule.cs" --exclude-attributes-list "C:\Users\angocke\code\runtime\eng\DefaultGenApiDocIds.txt" --header-file "C:\Users\angocke\code\runtime\eng\LicenseHeader.txt" --lang-version "latest" --follow-type-forwards" exited with code 3. [C:\Users\angocke\code\runtime\src\libraries\System.Runtime\tests\TestModule\System.Reflection.TestModule.ilproj]

The system cannot find the path specified.
```

Reply:
It seems like you ran it against the solution file. Try running it under the project src directory directly so that it runs on the src csproj only.

Reply:
OK, did that, and no errors, but it basically just deleted most of the System.Runtime.cs file.

Reply:
Also, will this fix the "seed" problem I mentioned below?

Reply:
> OK, did that, and no errors, but it basically just deleted most of the System.Runtime.cs file.

Hmm weird. Did you build against a locally built System.Private.CoreLib? Could you paste the command that it generated when invoking GenAPI? You can get it via a binlog.

> Also, will this fix the "seed assembly" problem I mentioned below?

That is a `GenFacades` issue. @ericstj or @Anipik would be able to help you best.

Reply:
>Also, will this fix the "seed assembly" problem I mentioned below?

Probably it is because you didn't built against an updated S.P.CoreLib which contains your new type.